test/unittest/dds/subscriber/DataReaderTests.cpp

```diff
@@ -2315,7 +2315,6 @@ class DataReaderUnsupportedTests : public ::testing::Test
 /*
  * This test checks that the DataReader methods defined in the standard not yet implemented in FastDDS return
  * ReturnCode_t::RETCODE_UNSUPPORTED. The following methods are checked:
- * 1. get_sample_lost_status
  * 2. get_sample_rejected_status
```

Comment:
Change numbers in list:

```suggestion
 * 1. get_sample_rejected_status
```